nautobot/ipam/migrations/0022_aggregate_to_prefix_data_migration.py

```diff
@@ -158,7 +158,11 @@ def migrate_aggregate_to_prefix(apps, schema_editor):
             prefix_length=instance.prefix_length,
             vrf__isnull=True,
         ).exists():
-            prefix = Prefix.objects.get(network=instance.network, prefix_length=instance.prefix_length)
+            prefix = Prefix.objects.filter(
+                network=instance.network,
+                prefix_length=instance.prefix_length,
+                vrf__isnull=True,
+            ).first()
```

Comment:
Is `first` correct/sufficient here or do we need to handle *all* matching Prefixes?

Reply:
This is trying to find a suitable prefix to migrate the aggregate fields to (date_allocated and rir). It's unlikely that someone has an aggregate with the same network/length as duplicate prefixes without a vrf. We'd probably want to tie in to the namespace migrations if we were going to try to account for that situation. The more likely scenario (which was encountered and prompted the bug report) is where they have an aggregate, a prefix without a vrf and a prefix with a vrf that all have the same network/length.

Reply:
Unlikely sure but without `ENFORCE_GLOBAL_UNIQUE=True` it's certainly possible in 1.x. Probably not worth losing sleep over though. Thanks.